Common/Gfx/Canvas.cpp

```diff
@@ -563,6 +570,29 @@ void Canvas::DrawMaskedBitmap(Gdiplus::Bitmap* bitmap, Gdiplus::Bitmap* maskBitm
 	bitmapLock->Release();
 }
 
+void Canvas::DrawGeometry(const Shape & shape, D2D1_MATRIX_3X2_F & transform)
+{
+	if (!BeginTargetDraw()) return;
+
+	m_Target->SetAntialiasMode(D2D1_ANTIALIAS_MODE_ALIASED);
```

Comment:
Could you do something like `const auto originalTransform = m_Target->GetAntialiasMode();` and use that in the `SetAntialiasMode` call at the end?